client/js/createPost.js

```diff
@@ -21,9 +19,22 @@ const createPost = () => {
     }).then((response) => {
         return response.json()
     }).then((data) => {
-        console.log('Message:', data.message)
+        // url2 output what process.env.VARIABLE is between BASE_URI & LOCALHOST in server/routes/db.js
+        const getServerUrl = (data) => {
+            return (data.baseUrl === 'https://gmshortener.herokuapp.com/') ?
```

Comment:
is there a reason you can't just return the baseUrl?

Reply:
It seems like baseUrl should be localhost on local, and gmshortener on prod, and both point to the same process.env

Reply:
Ahh, I misunderstood somewhere, but will fix it in the next commit.

Reply:
No problem. (: Just open a new issue and close the main one for deploying to heroku, this isn't a bad problem to have